packages/flutter/test/cupertino/route_test.dart

```diff
@@ -502,6 +502,165 @@ void main() {
     expect(tester.getTopLeft(find.byType(Placeholder)).dy, closeTo(600.0, 0.1));
   });
 
+  testWidgets('Bottom route has parallax when non fullscreen route is pushed on top', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      CupertinoApp(
+        home: Builder(
+          builder: (BuildContext context) {
+            return Column(
+              children: <Widget>[
+                const Placeholder(),
+                CupertinoButton(
+                  child: const Text('Button'),
+                  onPressed: () {
+                    Navigator.push<void>(context, CupertinoPageRoute<void>(
```

Comment:
you're not testing the new functionality on CupertinoFullscreenDialogTransition you created right? That would involve 2 routes with the bottom one having fullscreenDialog true

Reply:
You’re correct, it seems I forgot to also test that explicitely.
This test is still needed though since it was missing.  
Is it considered bad or good practice to move de nody of this test into a function that gets an argument? (Whether the bottomroute should be fuulscreenDialog)

Reply:
having helper functions is ok